website/docs/guides/best-practices/how-we-mesh/mesh-1-intro.md

```diff
@@ -6,19 +6,21 @@ hoverSnippet: Learn how to get started with dbt Mesh
 
 ## What is dbt Mesh?
 
-Organizations of all sizes rely upon dbt to manage their data transformations, from small startups to large enterprises. At scale, it can be challenging to coordinate all the organizational and technical requirements demanded by your stakeholders within the scope of a single dbt project. To date, there also hasn't been a first-class way to effectively manage the dependencies, governance, and workflows between multiple dbt projects.
+Organizations of all sizes rely upon dbt to manage their data transformations, from small startups to large enterprises. At scale, it can be challenging to coordinate all the organizational and technical requirements demanded by your stakeholders within the scope of a single dbt project.
 
-Regardless of your organization's size and complexity, dbt should empower data teams to work independently and collaboratively; sharing data, code, and best practices without sacrificing security or autonomy. dbt Mesh provides the tooling for teams to finally achieve this.
+To date, there also hasn't been a first-class way to effectively manage the dependencies, governance, and workflows between multiple dbt projects. 
 
-dbt Mesh is not a single product: it is a pattern enabled by a convergence of several features in dbt:
+That's where **dbt Mesh** comes in - empowering data teams to work *independently and collaboratively*; sharing data, code, and best practices without sacrificing security or autonomy. 
+
+This guide will walk you through the concepts and implementation details needed to get started. dbt Mesh is not a single product: it is a pattern enabled by a convergence of several features in dbt:
 
 - **[Cross-project references](/docs/collaborate/govern/project-dependencies#how-to-use-ref)** - this is the foundational feature that enables the multi-project deployments. `{{ ref() }}`s now work across dbt Cloud projects on Enterprise plans.
 - **[dbt Explorer](/docs/collaborate/explore-projects)** - dbt Cloud's metadata-powered documentation platform, complete with full, cross-project lineage.
 - **Governance** - dbt's new governance features allow you to manage access to your dbt models both within and across projects.
-  - **[Groups](/docs/collaborate/govern/model-access#groups)** - groups allow you to assign models to subsets within a project.
+  - **[Groups](/docs/collaborate/govern/model-access#groups)** - groups allow you to gather together nodes in your dbt DAG that are logically connected (such as by functional area) and assign an owner to the group.
```

Comment:
I don't __love__ this framing but I think it's better than what's currently there if anyone wants to take a go at it